test/unit/clients/connector-client/connector-post-worldpay-check-3ds-flex-config.pact.test.js

```diff
@@ -0,0 +1,127 @@
+            .withState('has the credentials')
+            .withUponReceiving('a request to check Worldpay 3DS Flex credentials')
+            .withMethod('POST')
+            .withRequestHeaders({ 'Content-Type': 'application/json' })
+            .withRequestBody(pactifiedRequest.payload)
+            .withStatusCode(200)
+            .withResponseHeaders({ 'Content-Type': 'application/json' })
+            .withResponseBody(pactifiedResponse)
+            .build()
+        ).then(() => done())
+          .catch(done)
+      })
+
+      it('should return valid', done => {
+        const valid3dsFlexCredentials = checkValidWorldpay3dsFlexCredentialsRequest.getPlain()
+        const validResult = checkValidWorldpay3dsFlexCredentialsResponse.getPlain()
+        connectorClient.postCheckWorldpay3dsFlexCredentials(valid3dsFlexCredentials)
+          .should.be.fulfilled.then((response) => {
+            expect(response).to.deep.equal(validResult)
+          }).should.notify(done)
+      })
+    })
+
+    describe('and the credentials are invalid and pass the existing format validation', () => {
+      const checkInvalidWorldpay3dsFlexCredentialsRequest = worldpay3dsFlexCredentialsFixtures.checkInvalidWorldpay3dsFlexCredentialsRequest()
+      const checkInvalidWorldpay3dsFlexCredentialsResponse = worldpay3dsFlexCredentialsFixtures.checkInvalidWorldpay3dsFlexCredentialsResponse()
+      before(done => {
+        const pactifiedRequest = checkInvalidWorldpay3dsFlexCredentialsRequest.getPactified()
+        const pactifiedResponse = checkInvalidWorldpay3dsFlexCredentialsResponse.getPactified()
+
+        provider.addInteraction(
+          new PactInteractionBuilder(`${ACCOUNTS_RESOURCE}/${EXISTING_GATEWAY_ACCOUNT_ID}/${CHECK_WORLDPAY_3DS_FLEX_CREDENTIALS}`)
+            .withState('does not have the credentials')
+            .withUponReceiving('a request to check Worldpay 3DS Flex credentials')
+            .withMethod('POST')
+            .withRequestHeaders({ 'Content-Type': 'application/json' })
+            .withRequestBody(pactifiedRequest.payload)
+            .withStatusCode(200)
+            .withResponseHeaders({ 'Content-Type': 'application/json' })
+            .withResponseBody(pactifiedResponse)
+            .build()
+        ).then(() => done())
+          .catch(done)
+      })
+
+      it('should return invalid', done => {
+        const invalid3dsFlexCredentials = checkInvalidWorldpay3dsFlexCredentialsRequest.getPlain()
+        const invalidResult = checkInvalidWorldpay3dsFlexCredentialsResponse.getPlain()
+        connectorClient.postCheckWorldpay3dsFlexCredentials(invalid3dsFlexCredentials)
+          .should.be.fulfilled.then((response) => {
+            expect(response).to.deep.equal(invalidResult)
+          }).should.notify(done)
+      })
+    })
+
+    describe('and the credentials are valid and pass the existing format validation but credentials check fails', () => {
```

Comment:
As per the reason above, this test should be removed.

Reply:
Done.